comps/guardrails/pii_detection/README.md

```diff
@@ -0,0 +1,100 @@
+# PII Detection Microservice
```

Comment:
can you add the full name of PII for beginners to understand this microservice? 

PS: it's better to update OPEA API spec as well to define this component's behavior